include/slang/ast/types/TypePrinter.h

```diff
@@ -25,6 +25,8 @@ struct SLANG_EXPORT TypePrintingOptions {
     bool addSingleQuotes = false;
     bool elideScopeNames = false;
     bool printAKA = false;
+    bool skipScopedTypeNames = false;
```

Comment:
Can you make sure these are exposed in the Python bindings as well? Can be easily added in the bindings/python/types.cpp file.

Reply:
Done